### ISSUE TITLE
Create new runbook(s) + update existing runbook(s) that call for a redirect request 

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#13109]
#### Background

The current redirect request runbook does not have a ticket or step to ensure the facility locator is updated via CSV change 

#### Acceptance Criteria 

- [x] I expect a new runbook is created in the form of a github issue template for URL changes
- [x] I expect the specific steps to create this new runbook are created with help desk input 
- [x] I expect it is determined if changes are needed to the existing redirect request runbook in partnership with help desk input 
- [ ] I expect all existing runbooks are reviewed and those that reference a redirect request are updated according the new runbook created for URL changes 

#### Implementation steps
I have identified the runbooks that need to change in [this Google Sheet](https://docs.google.com/spreadsheets/d/1PWYItEgg1th_LPnIDp4LLR14FgbttfAkJZ66DrTolxM/edit#gid=0).

**Comment by omahane** [comment]
@TroyA6  I have drafted a document for a new URL change runbook and would like to get your feedback:
https://github.com/department-of-veterans-affairs/va.gov-cms/blob/33a29ad1dac6d3f050084c13a5ffbd88ce3bbef8/.github/ISSUE_TEMPLATE/runbook-facility-url-change.md

I have also included changes in the other runbooks to reference this, but I want to make sure that this makes sense. It's essential the sibling to the redirect request, sometimes showing up with it and sometimes not. Thus, it's a runbook unto itself.

Let me know if this makes sense or not.
